packages/contracts/tasks/index.ts

```diff
@@ -0,0 +1,8 @@
+export * from './deploy'
+export * from './l2-gasprice'
+export * from './set-owner'
+export * from './validate-address-dictator'
+export * from './validate-chugsplash-dictator'
+export * from './whitelist'
+export * from './withdraw-fees'
+export * from './fetch-batches'
```

Comment:
Looks like there's just one more linting issue. You can automatically fix linting issues by running the command:

```
yarn lint:fix
```